scripts/start-relayer.sh

```diff
@@ -1,23 +1,34 @@
 #!/bin/bash
 
-RELAY_PATH="${RELAY_PATH:-local_testnet}"
+RELAY_PATH="${RELAY_PATH:=local_local2}"
+RELAY_CMD="${RELAY_CMD:-rly}"
+RELAYER_HOME="${RELAYER_HOME:-$HOME/.relayer}"
 
 # We have to get CHAIN_1 and CHAIN_2 by splitting the path
 CHAIN_1=$(echo "${RELAY_PATH}" | cut -d "_" -f 1)
 CHAIN_2=$(echo "${RELAY_PATH}" | cut -d "_" -f 2)
 
 check_keys() {
     CHAIN=$1
-    rly keys show $CHAIN &> /dev/null
+    ${RELAY_CMD} keys show $CHAIN --home ${RELAYER_HOME} &> /dev/null
```

Comment:
`$CHAIN` and `${RELAYER_HOME}` need double quotes.

Reply:
Agreed